### ISSUE TITLE
SUMMARY: Satisfactory

### ISSUE THREAD

**Issue by jzhu01** [middleduck/prj-rev-bwfs-dasmoto#6]
Great job on this project! I really liked your usage of IDs, and the organization of your html file - it was very easy to read. In terms of improvement, I recommend removing the font-weight:bold on your h1 and h2s, as well as deleting repetitive font-family stylings in the CSS.

If you’re in the mood for a challenge...I would look into adding more items under each section and finding a nice way to lay them out. Here's a link to get you started - https://www.w3schools.com/css/css_rwd_grid.asp. Keep up the awesome work! ^_^

**Comment by middleduck** [comment]
Hi Jen, thanks for the feedback.  Will definitely check out the link and try and incorporate more of that into the projects.  Out of curiosity, do we have to stick to the 'letter of the law' when doing the projects or can we add additional styling etc?

**Comment by jzhu01** [comment]
Of course! It's my pleasure ^_^ You definitely do not have to stick to the "letter of the law" when it comes to these projects. I'd recommend exploring with your creativity and seeing what else you can bring to the table. Happy coding! :)

**Comment by middleduck** [comment]
Hi Jen, hope you don't mind me asking here - struggling with adding a picture behind a heading.  Can you advise how I would do that?  I've put the heading in a <div> and referenced that <div> in the CSS to add a background image to it - but it's driving me nuts.  Can't do it.
